src/lua/widget/combobox.c

```diff
@@ -68,8 +68,6 @@ static int combobox_numindex(lua_State*L)
     return 0;
   }
   if(key <= 0 || key > length) {
-    return luaL_error(L,"Invalid index for combo box : %d\n",key);
-  } else if (key > length) {
```

Comment:
I think it's more consistent with regular lua behaviour - you can for example use ipair just like in regular table.

And this is the part of commit message for this:
> combobox fix: `for` with `ipairs` over combobox widget cause an error

However, I'll change this to be more accurate.